src/System.Diagnostics.StackTrace/tests/StackFrameTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public class StackFrameTests
+    {
+        [Fact]
+        public void OffsetUnknown_Get_ReturnsNegativeOne()
+        {
+            Assert.Equal(-1, StackFrame.OFFSET_UNKNOWN);
+        }
+
+        [Fact]
+        public void Ctor_Default()
+        {
+            var stackFrame = new StackFrame();
+            VerifyStackFrame(stackFrame, false, 0, typeof(StackFrameTests).GetMethod(nameof(Ctor_Default)));
```

Comment:
FYI: Reflection behaves differently in uapaot, so I'll run your tests in that platform just to make sure that they all pass.